### PR TITLE
second PR for [AMITT] add AMITT Tactics and Technique tags

### PR DESCRIPTION
**Pull request by yugoslavskiy**
@VVX7 has created a PR #151 with the next description:

```
Add support for rendering AM!TT Tactics and Techniques in Markdown and Confluence templates.
Add AM!TT mapping.
Add script to update AM!TT mapping from STIX2 repo.
```

We couldn't process it for a while, and now there are conflicts with `master` and `develop` branches, that I've fixed and now we can discuss the functionality, work it out and merge (:

### CODE REVIEW

File: scripts/update_amitt_mapping.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import json, os, requests
+
+amitt_tactic_mapping = {}
+amitt_technique_mapping = {}
+amitt_mitigation_mapping = {}
+
+amitt_json_url = ("https://raw.githubusercontent.com/VVX7/cti/master/amitt-attack/amitt-attack.json")
```
Comment (yugoslavskiy) [inline scripts/update_amitt_mapping.py]: Is there any official source of AMITT data? Uploading data from personal fork doesn't look reliable.

Comment (VVX7) [reply scripts/update_amitt_mapping.py]: Right. I'll move it to the official amitt repo shortly.

File: scripts/templates/markdown_responseplaybook_template.md.j2 [hunk 2]
```diff
@@ -3,6 +3,8 @@
...
 | ATT&amp;CK Tactic    | {%- if tactics is defined and tactics|length %}<ul>{% for tactic_name, tactic_id in tactics  %}<li>[{{tactic_id}}: {{tactic_name}}](https://attack.mitre.org/tactics/{{tactic_id}})</li>{% endfor %}</ul>{% else %} None{%endif%}  |
 | ATT&amp;CK Technique | {%- if techniques is defined and techniques|length %}<ul>{% for technique_name, technique_id in techniques  %}<li>[{{technique_id}}: {{technique_name}}](https://attack.mitre.org/tactics/{{technique_id}})</li>{% endfor %}</ul>{% else %} None{%endif%}                             |
+| AM&excl;TT Tactic    | {%- if amitt_tactics is defined and amitt_tactics|length %}<ul>{% for tactic_name, tactic_id in amitt_tactics  %}<li>[{{tactic_id}}: {{tactic_name}}](https://vvx7.xyz/tactics/{{tactic_id}})</li>{% endfor %}</ul>{% else %} None{%endif%}  |
+| AM&excl;TT Technique | {%- if amitt_techniques is defined and amitt_techniques|length %}<ul>{% for technique_name, technique_id in amitt_techniques  %}<li>[{{technique_id}}: {{technique_name}}](https://vvx7.xyz/tactics/{{technique_id}})</li>{% endfor %}</ul>{% else %} None{%endif%}                             |
```
Comment (yugoslavskiy) [inline scripts/templates/markdown_responseplaybook_template.md.j2]: @VVX7 could you please develop this template the same way we've done it for Data Needed?
This way if there is no value for a field, the line would not be rendered at all:

https://github.com/atc-project/atomic-threat-coverage/blob/ea6cda337fe91d09635193c7f854bd3a71437959/scripts/templates/markdown_dataneeded_template.md.j2#L4

This way, we wouldn't have redundant lines with "None" value in each and every markdown document.

Comment (VVX7) [reply scripts/templates/markdown_responseplaybook_template.md.j2]: Ok, this is the official AMITT stix repo: https://github.com/cogsec-collaborative/amitt_cti
I just need to add the STIX data to it.

I'll make that change so it's consistent with Data Needed.  Probably tomorrow.

Comment (yugoslavskiy) [reply scripts/templates/markdown_responseplaybook_template.md.j2]: As you can see, some of the fields implemented in the way to put there `None`. But these fields are common, and in most of cases there should be some value. A few months ago we've added `Mitigation Policies` field to Data Needed entities, and it was `None` for majority of entities, and that wasn't good.

Oh, totally forgot to do that for Detection Rules markdown template. Will do right now %)

Comment (yugoslavskiy) [reply scripts/templates/markdown_responseplaybook_template.md.j2]: I've updated Detection Rule and Data Needed templates. #159 closed now. So please do ```

**Comment by VVX7** [comment]
ATT&CK and AM!TT tactics/techniques are conditionally inserted in markdown playbook templates if the appropriate attack.x and amitt.x tags are present.

**Comment by VVX7** [comment]
I also modified the incident response phases to include language that's better suited for AM!TT.

**Pull request merged**